src/components/Navigation/components/Item/Item.tsx

```diff
@@ -266,6 +266,7 @@ export function Item({
   function getClickHandler(onClick: ItemProps['onClick']) {
     return (event: MouseEvent<HTMLElement>) => {
       const {currentTarget} = event;
+      currentTarget.blur();
```

Comment:
Blurring to remove the focus ring for nav items on click. Tab order remains so clicking and then tabbing will still bring focus to the next nav item

Reply:
This breaks keyboard navigation, removing focus from navigation items when the `enter` key is pressed. This'll have to go. It also breaks in the old design language, as this change isn't scoped to the new DL.

Why not apply the focus ring using the `:focus:not(:active)` selector instead? That should solve the problem without breaking accessibility. If not, we're going to have to live with the ring

|😢 |
|-|
|![broken-keyboard-navigation](https://user-images.githubusercontent.com/1403188/75565614-46b3e400-5a1c-11ea-9ced-0343f9dd7469.gif)|

Reply:
If that doesn't work, a keyboard focused state like used in checkbox will.